Model/Client/Transaction.php

```diff
@@ -76,6 +76,23 @@ public function refund($transaction_id, $amount = null)
         }
     }
 
+    /**
+     * void transaction
+     *
+     * @param string
+     * @return Array
+     */
+    public function void($transaction_id)
+    {
+        try {
+            return $this->getGr4vyConfig()->voidTransaction($transaction_id);
```

Comment:
Is this calling a function that doesn't exist yet in `gr4vy-php` or is this working now?

Reply:
Adding it here: https://github.com/gr4vy/gr4vy-php/pull/24, but yes, the gr4vy-php version will need to be updated

Reply:
Fixed